e2e-tests/portal/cypress/e2e/journeys/maker/dashboard/deals/dashboard-table-data.spec.js

```diff
@@ -179,16 +177,18 @@ context('View dashboard deals as a maker', () => {
 
   // TODO: DTFS2-5372 - fix.
   it('should not show deals created by other banks', () => {
-    cy.login(BANK1_MAKER1);
+    cy.login(ADMIN);
```

Comment:
I think this should be `BANK1_MAKER1` as it's not an admin dashboard test?

Reply:
With `BANK1_MAKER1` user assertion fails as number of the deals returned are 3 instead of 4.

![Capture](https://github.com/user-attachments/assets/13fa6186-d916-4a75-a3c6-6f779abc3f02)